frappe/public/js/frappe/form/controls/json.js

```diff
@@ -0,0 +1,15 @@
+frappe.ui.form.ControlJSON = frappe.ui.form.ControlCode.extend({
```

Comment:
@Kratos-Freyja Please add JSON validation. JSON control should check if the entered JSON is valid.